flow-typed/declarations.js

```diff
@@ -0,0 +1,10 @@
+/*
+  If Flow throws "Required module not found", altough module is not missing,
```

Comment:
@zepod https://github.com/flowtype/flow-typed is nice lib which is adding existing type definition instead of you. You can just run `flow-typed install` and It installs exist declared types or creates new (with "any") for not existing.